release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -177,6 +177,16 @@ For more information, see link:https://kubernetes.io/docs/concepts/services-netw
 
 [id="ocp-4-11-osdk"]
 === Operator development
+==== Performance Addon Operator functions moved to the Node Tuning Operator
+
+In earlier versions of {product-title}, the Performance Addon Operator was used to implement automatic tuning to achieve low latency performance for OpenShift applications. In {product-title} 4.11, these functions are part of the Node Tuning Operator. The Node Tuning Operator is part of the standard installation for {product-title} 4.11. If you upgrade to {product-title} 4.11, the Node Tuning Operator removes the Performance Addon Operator, and all related artifacts, on startup. 
+
+[NOTE]
+====
+You must still use the `performance-addon-operator-must-gather` image when running the `must-gather` command with the Performance Profile Creator.
```

Comment:
The `performance-addon-operator-must-gather` image is not referenced on the Node Tuning Operator page that is referenced on line 189. To provide the reader with further context about this note, you could add a cross-reference to https://docs.openshift.com/container-platform/4.10/scalability_and_performance/cnf-create-performance-profiles.html#gathering-data-about-your-cluster-using-must-gather_cnf-create-performance-profiles, or suchlike. For example:

> You must still use the `performance-addon-operator-must-gather` image when running the `must-gather` command with the Performance Profile Creator. For more information, see xref:../scalability_and_performance/cnf-create-performance-profiles.adoc#gathering-data-about-your-cluster-using-must-gather_cnf-create-performance-profiles[Gathering data about your cluster using must-gather].